docker/dev/keycloak/themes/openproject/account/resources/css/style.css

```diff
@@ -0,0 +1,32 @@
+:root body {
+  --pf-v5-global--primary-color--100: #1A67A3;
+  /*--pf-v5-global--primary-color--200: TODO...;*/
+}
+
+:root body .pf-v5-c-nav {
+  --pf-v5-c-nav__link--m-current--BackgroundColor: #175A8E;
+  --pf-v5-c-page__sidebar--BackgroundColor: #333739;
+}
+
+:root body .pf-v5-c-masthead {
+  --pf-v5-c-masthead--BackgroundColor: #1A67A3;
+}
+
+:root body .pf-v5-c-masthead .pf-v5-c-button {
+  --pf-v5-c-button--m-link--Color: white;
+  --pf-v5-c-button--m-link--m-inline--hover--Color: white;
+}
+
+:root body .pf-v5-c-masthead .pf-v5-c-toolbar {
+  --pf-v5-c-masthead--c-toolbar--BackgroundColor: #1A67A3;
+}
+
+body {
+  background-color: rgb(235, 243, 247);
+}
+
+.pf-v5-c-brand {
+  font-size: 2em;
```

Comment:
❓ any reason to use `em` instead of `rem`?

Do you want to override root fontsize?

Reply:
Totally makes sense. Mistake of someone not using CSS regularly enough.